redbot/cogs/general/general.py

```diff
@@ -202,54 +202,71 @@ def __init__(self):
         special_date = datetime.datetime(2016, 1, 10, 6, 8, 4, 443000)
         is_special = (user.id == 96130341705637888 and
                       guild.id == 133049272517001216)
-        
-        roles = sorted(user.roles)[1:]
-
-        joined_at = user.joined_at if not is_special else special_date
-        since_created = (ctx.message.created_at - user.created_at).days
-        since_joined = (ctx.message.created_at - joined_at).days
+        bot = self.bot
+        realname = str(user)
+        useravatarurl = user.avatar_url
+        joined_at = self.fetch_joined_at(user, server)
+        since_created = (ctx.message.timestamp - user.created_at).days
+        since_joined = (ctx.message.timestamp - joined_at).days
         user_joined = joined_at.strftime("%d %b %Y %H:%M")
         user_created = user.created_at.strftime("%d %b %Y %H:%M")
-        member_number = sorted(guild.members,
+        member_number = sorted(server.members,
```

Comment:
Why'd you make this change? in V3 it is `guild` and `guild` is being used elsewhere in this command.

Reply:
Don't actually remember making that change. I was trying to convert what i had written in v2 to to v3 because v2 is feature locked. It could be because of a little copy paste mistake.